locale/fi/LC_MESSAGES/django.po

```diff
@@ -627,36 +647,26 @@ msgstr "Valittu venepaikka ei ole aktiivinen"
 msgid "place"
 msgstr "paikka"
 
-#, fuzzy
-#| msgid "Inspection"
 msgid "section"
-msgstr "Tarkastus"
+msgstr "osa-alue"
 
-#, fuzzy
-#| msgid "order number"
 msgid "sticker number"
-msgstr "tilausnumero"
+msgstr "tarran numero"
 
-#, fuzzy
-#| msgid "order number"
 msgid "sticker posted"
-msgstr "tilausnumero"
+msgstr ""
 
 msgid "winter storage lease"
 msgstr "talvisäilytyspaikan vuokrasopimus"
 
 msgid "winter storage leases"
 msgstr "talvisäilytyspaikkojen vuokrasopimukset"
 
-#, fuzzy
-#| msgid "Lease cannot have both place and area assigned"
 msgid "Lease cannot have both place and section assigned"
-msgstr "Vuokrasopimukseen ei voida liittää sekä paikkaa että aluetta"
+msgstr "Vuokrasopimukseen ei voida liittää sekä paikkaa että osa-aluetta"
```

Comment:
Not 100% sure if this is the term for "section"

Reply:
In other places where "section" was used, it was translated as "osa-alue". Not sure if it's correct but it's consistent :grin: